GITFLOW.md

```diff
@@ -27,56 +27,56 @@ git commit -m "second commit"
 # rebase against staging to pull in any changes that have been made
 # since you started your feature branch.
 git fetch
-git rebase origin/develop
+git rebase origin/master
 
 # push your local changes up to the remote
 git push
 
 # if you've already pushed changes and have rebased, your history has changed
 # so you will need to force the push
 git fetch
-git rebase origin/develop
+git rebase origin/master
 git push --force-with-lease
 ````
 
 
 ### GitHub workflow
 
-- Open a Pull Request against `staging`. Check our PR guidelines [here](https://github.com/quintoandar/butterfree/blob/master/CONTRIBUTING.md#pull-request-guideline).
+- Open a Pull Request against `master`. Check our PR guidelines [here](https://github.com/quintoandar/butterfree/blob/master/CONTRIBUTING.md#pull-request-guideline).
 - When the Pull Request has been approved, merge using `squash and merge`, adding the Jira task number and a brief description:
 ie, `[MLOP-169] Enable stream pipelines in Butterfree`.
```

Comment:
We should change replace this reference to Jira with Github issues. (Internally we can keep using Jira, but to embrace external contributors this should mention issues)